### DISCUSSION TITLE
What is the best way to initialize formData in @rjsf/material-ui-v5

### DISCUSSION THREAD

**Discussion by dagda1** [rjsf-team/react-jsonschema-form#3243] [Q&A]
What is the best way to initialize form data in `@rjsf/core-v5`?

If I pass 3 fields that are undefined when the form loads, how can I get them into the `formData`?

If not, is the only way to use `formData`?

```tsx
<Form
   formData={{
     field1: undefined,
     field2: undefined,
     field3: undefined
```

Has this behaviour changed between v4 and v5?

**Comment by aress31** [comment]
Bump, I am interested to know too, additionally how to initialise specific fields, using, should one use the `onChange`?
